xcode/Subconscious/Shared/Components/Common/MarkupTextViewRepresentable.swift

```diff
@@ -123,16 +123,34 @@ struct MarkupTextViewRepresentable: UIViewRepresentable {
                 )
             )
         }
+        
+        // This allows us to intercept touches on embedded transclude blocks
+        override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+            guard let touch = touches.first else { return }
+            let tapPoint = touch.location(in: self)
+            
+            let textContentStorage = self.textLayoutManager?.textContentManager as! NSTextContentStorage
+            
+            if let layoutFragment = textLayoutManager!.textLayoutFragment(for: tapPoint) {
+                let content = textContentStorage.attributedString(for: layoutFragment.textElement!)
+                
+                // TODO: check for whether this tap should navigate to a link
+                MarkupTextViewRepresentable.logger.debug("Tapped: \(String(describing: content?.string))")
+                // Calling super preserves default behaviour
+                super.touchesBegan(touches, with: event)
+            }
+        }
     }
 
     //  MARK: Coordinator
-    class Coordinator: NSObject, UITextViewDelegate, NSTextStorageDelegate {
+    class Coordinator: NSObject, UITextViewDelegate, NSTextContentStorageDelegate, NSTextContentManagerDelegate, NSTextLayoutManagerDelegate {
         /// Is event happening during updateUIView?
         /// Used to avoid setting properties in events during view updates, as
         /// that would cause feedback cycles where an update triggers an event,
         /// which triggers an update, which triggers an event, etc.
         var isUIViewUpdating: Bool
         var representable: MarkupTextViewRepresentable
+        var renderTranscludeBlocks: Bool = false
```

Comment:
Gate rendering of embedded transclude blocks, they work / render mostly as expected but there's too many rough edges to fix for this initial PR.